### DISCUSSION TITLE
how to generate a single file with custom html and css files

### DISCUSSION THREAD

**Discussion by SusejR** [allure-framework/allure2#2532] [Questions & Support]
Hello, I am trying to generate a custom single-file with the results of my tests, I have tried to generate the report with the command "allure generate allure-results -o allure-report" and after generating it, I have tried to replace the index.html files , styles.css and app.js, once that is done I can open it without problem with the "allure open" command but I have not found information on how to convert that into a single html file that takes into account the test results but remains customized with my html, css and js files, thank you very much for your help

**Comment by seafat1203** [comment]
Hello I'm using this: https://github.com/MihanEntalpo/allure-single-html-file
and it works for me to generate some single html file to share with my clients. 
Hope it works for you.
